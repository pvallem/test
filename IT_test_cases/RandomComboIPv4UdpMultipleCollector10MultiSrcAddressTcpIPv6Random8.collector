bidir_flow_info {
  key {
    src_address: "\001\002\003\001"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\026\r\245d\314\023\304d\226\363U\321\304\347c\""
    dst_address: "\240\241e\273\026\251\t\007\313\232b\257v\237\212\236"
    src_port: 3754
    dst_port: 35851
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\000"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\002"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000040
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "S\361\324kd\253\341e\343\256\341\354\313\313\254l"
    dst_address: "\346\233g\202\271\354\310w\244\222\316\226-=\371e"
    src_port: 13737
    dst_port: 18981
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
