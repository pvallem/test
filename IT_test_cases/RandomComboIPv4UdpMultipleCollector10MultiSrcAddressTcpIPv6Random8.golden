flow_info {
  key {
    src_address: "\001\002\003\000"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\240\241e\273\026\251\t\007\313\232b\257v\237\212\236"
    dst_address: "\026\r\245d\314\023\304d\226\363U\321\304\347c\""
    src_port: 35851
    dst_port: 3754
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "\001\002\003\001"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\346\233g\202\271\354\310w\244\222\316\226-=\371e"
    dst_address: "S\361\324kd\253\341e\343\256\341\354\313\313\254l"
    src_port: 18981
    dst_port: 13737
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: "\001\002\003\002"
    dst_address: "\001\002\003d"
    src_port: 12345
    dst_port: 12345
    proto: 17
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  start_time: 1388563200000040
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
